test/jdk/jdk/incubator/vector/LoadSvmlTest.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2020, 2021, Oracle and/or its affiliates. All rights reserved.
```

Comment:
I think the copyright year should be 2021 only since it's a new file, right?

Reply:
Thanks for catching it. I will correct the copyright year.